stravalib/tests/unit/test_client_utils.py

```diff
@@ -54,3 +57,34 @@ def test_params(self):
         self.assertEqual(self.get_url_param(url, "client_id"), "1")
         self.assertEqual(self.get_url_param(url, "redirect_uri"), "www.example.com")
         self.assertEqual(self.get_url_param(url, "approval_prompt"), "auto")
+
+
+class TestClientUploadActivity(TestBase):
+    client = Client()
+
+    def test_upload_activity_file_with_different_types(self):
+        """
+        Test uploading an activity.
+
+        NOTE: This requires clearing out the uploaded activities from configured
+        writable Strava acct.
```

Comment:
Is this statement true, given that `ApiV3.post` is mocked? I guess the Strava API is not hit?

Reply:
I forgot to remove the statement. I will remove it.